presto-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3FileSystem.java

```diff
@@ -185,6 +191,7 @@
     private long multiPartUploadMinPartSize;
     private PrestoS3AclType s3AclType;
     private boolean skipGlacierObjects;
+    private boolean requesterPaysEnabled;
```

Comment:
Nit: the `enabled` suffix is redundant for a boolean variable. Name this `requestorPays`